RedHatInsights/compliance-frontend
# feat: RHICOMPL-2602 - Enable column management by default

This will enable column management for all tables, but the `SystemsTable`.

**How to test:**

1. Ensure `localStorage` is cleared and does not set the feature flag
2. Go to any page with a table, for example the "SCAP Policies"
3. Verify that there is a action menu and "Manage Columns" is present and works properly.


## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
